modules/architecture-machine-roles.adoc

```diff
@@ -40,7 +40,12 @@ The `kubelet` service must not be newer than `kube-apiserver`, and can be up to
 [id="defining-workers_{context}"]
 == Cluster workers
 
-In a Kubernetes cluster, the worker nodes are where the actual workloads requested by Kubernetes users run and are managed. The worker nodes advertise their capacity and the scheduler, which is part of the master services, determines on which nodes to start containers and pods. Important services run on each worker node, including CRI-O, which is the container engine, Kubelet, which is the service that accepts and fulfills requests for running and stopping container workloads, and a service proxy, which manages communication for pods across workers.
+In a Kubernetes cluster, the worker nodes are where the actual workloads requested by Kubernetes users run and are managed. The worker nodes advertise their capacity and the scheduler, which is part of the master services, determines on which nodes to start containers and pods. Important services run on each worker node, including CRI-O, which is the container engine; Kubelet, which is the service that accepts and fulfills requests for running and stopping container workloads; a service proxy, which manages communication for pods across workers; and the runC or crun (Technology Preview) low-level container runtime, which creates and runs containers.
```

Comment:
Are we using _master_ here? In other places, we've excised this in favor of other words, such as _control panel_.

Reply:
I'd actually reverse this, because containers run with pods:

> nodes to start containers and pods.

> nodes to start pods and containers.